test/arbitrable-permission-list.js

```diff
@@ -0,0 +1,634 @@
+  //   const challengerTx = await arbitrablePL.challengeRegistration(ITEM_VALUE, {
+  //     from: challenger,
+  //     value: stake + arbitrationFee,
+  //     gasPrice: gasPrice
+  //   });
+  //   const challengerTxFee = challengerTx.receipt.gasUsed * gasPrice;
+
+  //   assert.equal(
+  //     await web3.eth.getBalance(submitter).toString(),
+  //     submitterBalBeforeReqCancel
+  //       .plus(arbitrationFee)
+  //       .minus(submitterTxFee)
+  //       .toString(),
+  //     'Submitter has not received the refund appropriately'
+  //   );
+  //   assert.equal(
+  //     await web3.eth.getBalance(challenger).toStrin(),
+  //     challengerBalBeforeReqCancel
+  //       .plus(arbitrationFee)
+  //       .minus(challengerTxFee)
+  //       .toString(),
+  //     'Challenger not received refund properly'
+  //   );
+  // });
+  it("Should create a dispute when someone challenges clearing ", async () => {
+    const centralizedArbitrator = await CentralizedArbitrator.new(
+      arbitrationFee,
+      { from: arbitrator }
+    );
+    const arbitrablePL = await ArbitrablePermissionList.new(
+      centralizedArbitrator.address,
+      arbitratorExtraData,
+      metaEvidence,
+      blacklist,
+      appendOnly,
+      rechallengePossible,
+      stake,
+      timeToChallenge,
+      { from: submitter }
+    );
+
+    await arbitrablePL.requestRegistration(ITEM_VALUE, {
+      from: submitter,
+      value: stake + arbitrationFee
+    });
+    const reqRegistrationItem = await arbitrablePL.items(ITEM_VALUE);
+    assert.equal(
+      reqRegistrationItem[0].toNumber(),
+      ITEM_STATUS.SUBMITTED,
+      "Item registeration request not submitted."
+    );
+
+    await arbitrablePL.executeRequest(ITEM_VALUE, {
+      from: submitter
+    });
+    let exeRequestItem = await arbitrablePL.items(ITEM_VALUE);
+    assert.equal(
+      exeRequestItem[0].toNumber(),
+      ITEM_STATUS.REGISTERED,
+      "Item is not registered"
+    );
+    await arbitrablePL.requestClearing(ITEM_VALUE, {
+      from: submitter,
+      value: stake + arbitrationFee
+    });
+    const reqClearingItem = await arbitrablePL.items(ITEM_VALUE);
+    assert.equal(
+      reqClearingItem[0].toNumber(),
+      ITEM_STATUS.CLEARING_REQUESTED,
+      "Item not in clearing requested"
+    );
+    await arbitrablePL.challengeClearing(ITEM_VALUE, {
+      from: challenger,
+      value: stake + arbitrationFee
+    });
+
+    const dispute = await centralizedArbitrator.disputes(0);
+    assert.equal(
+      dispute[0],
+      arbitrablePL.address,
+      "Arbitrable not set up properly"
+    );
+    assert.equal(
+      dispute[1].toNumber(),
+      2,
+      "Number of choices not set up properly"
+    );
+    assert.equal(
+      dispute[2].toNumber(),
+      arbitrationFee,
+      "Fee not set up properly"
+    );
+  });
+
+  // it('Should not be able to challenge clearing if arbitration fee has increased', async () => {
```

Comment:
Same as with the challengeRegistration test. Call `centralizedArbitratior.setArbitrationPrice` and pass it a higher value than item.balance.